algorithms/sort/radix_sort.py

```diff
@@ -0,0 +1,26 @@
+'''
+radix sort
+complexity: O(nk) . n is the size of input list and k is the digit length of the number
+'''
+def radix_sort(arr):
+    is_done = False
+    position = 1
+
+    while not is_done:
+        queue_list = [list() for _ in range(10)]
+        is_done = True
+
+        for num in arr:
+            digit_number = (int)(num/position)%10
```

Comment:
@aig031 Typecasting in Python is different from the C version (that you used here).
The correct syntax for typecasting is 
```python
digit_number = int(num / position) % 10
```
But the recommended way for this in Python _3_ is
```python
digit_number = num // position % 10
```
Also notice the spaces around the operators, for readability.